### PR TITLE
Emit channel name with slack socket event

### PR DESCRIPTION
**Pull request by begizi**
use an empty string when the channel is not found. ie for direct
messages to the bot

### CODE REVIEW

File: api/src/helios/slack/slack.go [hunk 1]
```diff
@@ -9,6 +9,7 @@ import (
 )
 
 var MessageChann chan interface{}
+var api *slack.Slack
```
Comment (samccone) [inline api/src/helios/slack/slack.go]: why this change?

Comment (begizi) [inline api/src/helios/slack/slack.go]: Saving the slack.Slack type to a package global so I can reference it from the different functions without passing it around.

File: api/src/helios/slack/slack.go [hunk 2]
```diff
@@ -30,7 +31,7 @@ func initSlackRTM() error {
 
 	chReceiver := make(chan slack.SlackEvent)
 
-	api := slack.New(token)
+	api = slack.New(token)
```
Comment (begizi) [inline api/src/helios/slack/slack.go]: @samccone This line is related. Once we have the api type setup we save it to that package global

File: api/src/helios/slack/slack.go (hunk 1 above)
Comment (samccone) [inline api/src/helios/slack/slack.go]: sometimes passing things around is nice :wink: 

but this lgtm

File: api/src/helios/slack/slack.go (hunk 2 above)
Comment (samccone) [inline api/src/helios/slack/slack.go]: yepppp

File: api/src/helios/slack/slack.go (hunk 1 above)
Comment (begizi) [inline api/src/helios/slack/slack.go]: It was clean enough to pass the slack api instance in. Changed. :dancers:

**Comment by joelhawksley** [comment]
@begizi do we want to support direct messages to the bot?

**Comment by begizi** [comment]
Direct messages would just send as `""` for the channel name

**Comment by joelhawksley** [comment]
@begizi I'm wondering if we even want to forward them at all. It would be easier to handle on the client if we were guaranteed a channel name. Maybe we set the default channel name to be "direct"?

**Comment by samccone** [comment]
no one is consuming it yet, we can iterate on it :goat:

**Comment by joelhawksley** [comment]
@samccone agreed- not a blocker, just a thought

**Comment by samccone** [comment]
#agile-hacks 👍👍🐴✈️🐑🚒😎🚨
